### PR TITLE
Documented new HSTS feature

### LINKED ISSUE #1299: HSTS Header out of the box for HTTPS pages
Contao automatically sends the `Strict-Transport-Security` header for Contao pages if you enabled `https` as protocol.

Add maybe here: https://docs.contao.org/manual/en/layout/site-structure/configure-pages/#further-settings-for-website-roots

### PR DESCRIPTION
**Pull request by Toflar**
Fixes https://github.com/contao/docs/issues/1299.

Unfortunately, the short code seems to create a block element so it creates an additional `<p>` element and duplicates the styling:

Bildschirmfoto 2024-02-02 um 17 28 47

Any way to fix this? /cc @fritzmg

### CODE REVIEW

**Comment by fritzmg** [comment]
Hm, which Hugo version are you using?

**Comment by Toflar** [comment]
`hugo v0.117` - just updated to 0.122, no difference.

**Comment by fritzmg** [comment]
Not sure why this happens. While newer Hugo versions are supposed to have better handling regarding the automatic wrapping of shortcode contents, nothing worked to prevent this - and it does not seem to make much sense anyway. May be some one on the forum can shed some light on it: https://discourse.gohugo.io/t/shortcode-without-markdown-nested-in-shortcode-with-markdown-is-wrapped-with-p/48080

**Comment by fritzmg** [comment]
It's fixed now :)

File: docs/manual/site-structure/pages-as-central-elements/website-root.en.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ automatically redirected to the corresponding website root of a website.
 **Protocol:** If your website is reachable via HTTPS then this setting must be configured accordingly. Your visitors will
 be automatically redirected to HTTPS.
 
+{{% notice tip %}}
+{{< version-tag "5.3" >}} If you select HTTPS, Contao will also automatically send the `Strict-Transport-Security`
+(HSTS) HTTP header, making your site even more secure. Of course, this does not apply to everything that is not delivered by Contao, but is delivered directly via the web server.
```
Comment (fritzmg) [inline docs/manual/site-structure/pages-as-central-elements/website-root.en.md]:
```suggestion
(HSTS) HTTP header, making your site even more secure. Of course this only applies to resources delivered by Contao, but not to anything that is delivered directly via the web server for example.
```

File: docs/manual/site-structure/pages-as-central-elements/website-root.de.md [hunk 2]
```diff
@@ -57,6 +57,12 @@ er automatisch zu dem entsprechenden Startpunkt einer Webseite weitergeleitet.
 **Protokoll:** Wenn deine Webseite über HTTPS verfügbar ist, muss diese Einstellung entsprechend konfiguriert werden. 
 Die Besucher werden automatisch auf HTTPS weitergeleitet.
 
+{{% notice tip %}}
+{{< version-tag "5.3" >}} Wenn du HTTPS auswählst, dann sendet Contao auch automatisch den `Strict-Transport-Security`
+(HSTS) HTTP Header und macht so deine Seite noch sicherer. Das gilt natürlich nicht für alles, was nicht von Contao,
+sondern direkt via Webserver ausgeliefert wird.
```
Comment (fritzmg) [inline docs/manual/site-structure/pages-as-central-elements/website-root.de.md]:
```suggestion
(HSTS) HTTP Header und macht so deine Seite noch sicherer. Das gilt natürlich nur für Ressourcen, die direkt von Contao geliefert werden, also z.B. nicht für alles, was direkt über den Webserver ausgeliefert wird.
```

**Review by fritzmg — COMMENTED** [review]
Some wording changes, wdyt?

**Review by netzarbeiter — APPROVED** [review]
Thank you, Yanick

**Pull request merged**